src/object/platform.cpp

```diff
@@ -156,6 +157,14 @@ Platform::editor_delete()
   GameObject::editor_delete();
 }
 
+void
+Platform::on_flip(float height)
+{
+  if (Path* const path = get_path()) {
```

Comment:
You have used `Path* const` at a few places (I won't mark them all as I usually do), which is a const pointer to a non-const object; is there a particular reason to do it this way?